spec/schema.rb

```diff
@@ -74,6 +74,20 @@
     t.column :project_alias_id, :integer
   end
 
+  create_table :specify_primary_key_models, force: true, partition_key: :account_id, primary_key: :entity_id do |t|
+    t.column :account_id, :integer
+    # t.column :entity_id, :integer (Excluded - would be a duplicate column since entity_id is already the primary key.)
+    t.column :name, :string
+  end
```

Comment:
Regarding the Rails specification, it was confusing that the syntax for specifying columns within the block differs depending on whether you explicitly specify a single symbol or an array for the `primary_key` parameter.